admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -145,6 +144,36 @@ this:
 
 .. _db-postgresql-label:
 
+Configure MySQL for 4-byte Unicode Support
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For supporting such features as emoji, you have to both enable 4-byte Unicode support in MySQL (instead of the default 3) as well as in ownCloud. 
+To do so:
+
+1. In ``config/config.sample.php`` set ``mysql.utf8mb4`` to true
```

Comment:
this is not necessary - during setup we are detecting if mb4 is possible .... I'm sorry for not providing the correct information.
Let me double check config.sample.php - maybe this needs some adoptions as well.